SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestSingleServerDynamicComponents/SpatialTestSingleServerDynamicComponents.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestSingleServerDynamicComponents.h"
+#include "TestDynamicComponentActor.h"
+#include "TestDynamicComponent.h"
+
+#include "Net/UnrealNetwork.h"
+#include "Kismet/GameplayStatics.h"
+
+/**
+ * This test tests dynamic component creation, attachment, removal and replication of properties in a single-server context.
+ *
+ * The test includes 1 Server and 2 Clients.
+ * The flow is as follows:
+ * - Setup:
+ *	- The TestActor is spawned and a dynamic component is immediately created and attached to it.
```

Comment:
```suggestion
 *  - The TestActor is spawned and a dynamic component is immediately created and attached to it.
```
?

Reply:
Hmm, it's always the first one that gets a tab...

Reply:
Do you have the clang-format thing set up?

Reply:
I think I do yeah, I have run the Setup.bat script and AFAIK that should do it, right?

Reply:
Is weird that I can't even see that there's a tab, I see it as a 1 space if that makes any sense?

Reply:
that does one part, you can also hook it up to Visual Studio directly, should be somewhere in Slack

Reply:
Oh, yes I just found a doc now.